app/cloud.go

```diff
@@ -175,3 +175,28 @@ func (a *App) SendNoCardPaymentFailedEmail() *model.AppError {
 	}
 	return nil
 }
+
+func (a *App) SendCloudTrialEndWarningEmail(trialEndDate, siteURL string) *model.AppError {
+	sysAdmins, e := a.getSysAdminsEmailRecipients()
```

Comment:
For assumed convention 
```suggestion
	sysAdmins, err := a.getSysAdminsEmailRecipients()
```